keras_cv/models/segmentation/deeplab.py

```diff
@@ -180,39 +177,42 @@ def get_config(self):
             "spatial_pyramid_pooling": self.spatial_pyramid_pooling,
             "segmentation_head": self.segmentation_head,
             "segmentation_head_activation": self.segmentation_head_activation,
+            "weight_decay": self.weight_decay,
         }
 
 
-@tf.keras.utils.register_keras_serializable(package="keras_cv")
+@keras.utils.register_keras_serializable(package="keras_cv")
 class SegmentationHead(layers.Layer):
     """Prediction head for the segmentation model
 
     The head will take the output from decoder (eg FPN or ASPP), and produce a
     segmentation mask (pixel level classifications) as the output for the model.
 
     Args:
-        num_classes: int, the number of output classes for the prediction. This should
+        num_classes: int. Number of output classes for the prediction. This should
```

Comment:
We've been using commas to separate the type from the description (e.g. in `output_scale_factor` below) -- should we be using periods instead throughout?

Reply:
No strong opinion as long as it's consistent (ideally, across all of our packages). I reverted the docstring to use commas.